src/bin/ldns.rs

```diff
@@ -0,0 +1,17 @@
+use std::process::ExitCode;
+
+use dnst::try_ldns_compatibility;
+
+fn main() -> ExitCode {
+    let mut args = std::env::args_os();
+    args.next().unwrap();
+    let args = try_ldns_compatibility(args).expect("ldns commmand is not recognized");
+
+    match args.execute() {
```

Comment:
I'm a little uncomfortable with the exit code handling here, shouldn't this be common to both `src/main.rs` and `src/bin/ldns.rs`? In fact I was expecting the whole of `src/bin/ldns.rs::main()` to be even thinner, e.g. something like"
```rust
fn main() -> ExitCode {
    let mut args = std::env::args_os();
    args.next().unwrap();
    app(args)
}
```

Reply:
Ok, I missed something @tertsdiepraam pointed out that my approach would also again let you run `dnst` commands which is a bit odd. The risk is low of this code changing often and if it becomes outdated compare to `src/main.rs` the testing would be impacted, not the actual application code so ... ignore me.